content/working_with/manager/ldap-integration.md

```diff
@@ -6,56 +6,177 @@ weight: 1450
 ---
 {{< param product_name >}} provides a user management mechanism, so you can define different users with different permissions, and upon login perform authentication and authorization to control the users’ access to resources. 
 
-The users can be either defined and managed in {{< param product_name >}} itself, or you can configure your Manager to integrate with an LDAP-based user-management system.
-You must select one of these options, as you cannot do both, and you must configure your manager accordingly upon installation or immediately afterwards, when no actions were performed on it yet.  
+The users can be either defined and managed in {{< param product_name >}} itself, or you can configure your Manager to integrate with an LDAP-based user-management system. You must select one of these options, as you cannot do both.
```

Comment:
```suggestion
The users can be either defined and managed in {{< param product_name >}} itself, or you can configure your {{< param cfy_manager_name >}} to integrate with an LDAP-based user-management system. You must select one of these options, as you cannot do both.
```

Reply:
Manager references updated in upcoming commit